gapic-generator-java/src/main/java/com/google/api/generator/gapic/protowriter/Writer.java

```diff
@@ -167,8 +181,12 @@ private static void writeSamples(
     }
   }
 
-  private static String writePackageInfo(
+  @VisibleForTesting
+  static String writePackageInfo(
       GapicPackageInfo gapicPackageInfo, JavaWriterVisitor codeWriter, JarOutputStream jos) {
+    if (gapicPackageInfo.isEmpty()) {
+      return "";
+    }
```

Comment:
nit: Can this check be done in the calling method? That way this doesn't need to return `""`? It just won't call `writePackageInfo` is the packageInfo object is empty.

Reply:
Done